calculator.rb

```diff
@@ -0,0 +1,22 @@
+def calculation(method, num1, num2)
```

Comment:
I'd also recommend rewriting the calculator as a class as with the example above :)